src/Sylius/Bundle/CoreBundle/Fixture/Factory/PaymentMethodExampleFactory.php

```diff
@@ -87,7 +90,8 @@ public function create(array $options = [])
             $paymentMethod->setCurrentLocale($localeCode);
             $paymentMethod->setFallbackLocale($localeCode);
 
-            $paymentMethod->setName(sprintf('[%s] %s', $localeCode, $options['name']));
+            $paymentMethod->setName($options['name']);
```

Comment:
Sounds like an idea, but I'd like to remove it now and bring back as soon as we start working on i18n scenarios for new shop.